here-naksha-lib-model/src/commonMain/kotlin/naksha/model/TagsEncoding.kt

```diff
@@ -9,24 +9,24 @@ import kotlin.js.JsExport
 @JsExport
 object TagsEncoding : FlagsBits() {
     /**
-     * A standard [JSON](https://datatracker.ietf.org/doc/html/rfc8259) encoded value.
+     * A JSON encoded using [JBON](https://github.com/heremaps/naksha/blob/v3/docs/JBON.md) encoding.
      */
-    const val JSON = 0 shl TAGS_SHIFT
+    const val JBON = 0 shl TAGS_SHIFT
 
     /**
-     * A standard [JSON](https://datatracker.ietf.org/doc/html/rfc8259) encoded value, compressed using
+     * A JSON encoded using [JBON](https://github.com/heremaps/naksha/blob/v3/docs/JBON.md) encoding, compressed using
      * [GZIP](https://en.wikipedia.org/wiki/Gzip).
      */
-    const val JSON_GZIP = 1 shl TAGS_SHIFT
+    const val JBON_GZIP = 1 shl TAGS_SHIFT
 
     /**
-     * A JSON encoded using [JBON](https://github.com/heremaps/naksha/blob/v3/docs/JBON.md) encoding.
+     * A standard [JSON](https://datatracker.ietf.org/doc/html/rfc8259) encoded value.
      */
-    const val JBON = 2 shl TAGS_SHIFT
+    const val JSON = 2 shl TAGS_SHIFT
```

Comment:
I hope that we always used references and not hard-coded values for these encodings - otherwise, we might end up surprised (JBON and JSON swapped) 😄

Reply:
In the SQL code uses hard-coded values (because otherwise we need to pay for a `plv8` call, just to read a constant), which exactly lead to the problem, that the tag decoding differs from feature decoding. As tag decoding right now was anyway broken, I just ensured that features and tags are in-sync and that the SQL code is as well simply the same.